src/Umbraco.Web/Trees/FormsTreeController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Net.Http.Formatting;
+using Umbraco.Web.Models.Trees;
+using Umbraco.Web.Mvc;
+using Umbraco.Web.WebApi.Filters;
+using Constants = Umbraco.Core.Constants;
+
+namespace Umbraco.Web.Trees
+{
+    [UmbracoTreeAuthorize(Constants.Trees.Forms)]
+    [Tree(Constants.Applications.FormsInstaller, Constants.Trees.Forms, SortOrder = 0, IsSingleNodeTree = true)]
+    [PluginController("UmbracoTrees")]
+    [CoreTree]
+    public class FormsTreeController : TreeController
```

Comment:
Do we even need the tree controller? We can have a section without a tree and it will just load in the dashboard

Reply:
I tried, but a red error `404` was shown.  How to avoid that if not by making an empty tree